frontend/src/views/pages/dealerships/Dealerships.vue

```diff
@@ -0,0 +1,176 @@
+                  <CBadge
+                    color="primary"
+                    v-if="
+                      $store.state.auth.createDealershipCompleted &&
+                      !$store.state.auth.createUserCompleted
+                    "
+                    :style="{
+                      'vertical-align': 'middle',
+                      'margin-top': '-0.5em',
+                    }"
+                    >NEW!</CBadge
+                  >
+                </td>
+              </template>
+            </CDataTable>
+          </CCardBody>
+        </CCard>
+      </CCol>
+    </CRow>
+    <CModal
+      :show.sync="addingDealership"
+      :centered="true"
+      title="Modal title 2"
+      size="lg"
+    >
+      <dealership-add />
+      <template #header>
+        <h6 class="modal-title">Create a new dealership!</h6>
+        <CButtonClose @click="addingDealership = false" />
+      </template>
+      <template #footer>
+        <!-- <CRow class="justify-content-center">
+        </CRow>
+        <CButton @click="addingDealership = false" color="danger">Discard</CButton>
+        <CButton @click="addingDealership = false" color="success">Accept</CButton> -->
+        <span></span>
+      </template>
+    </CModal>
+  </div>
+</template>
+
+<script>
+import Dealership from "./Dealership";
+import DealershipAdd from "./DealershipAdd.vue";
+
+const axios = require("axios");
+
+export default {
+  name: "Dealerships",
+  components: {
+    Dealership,
+    "dealership-add": DealershipAdd,
+  },
+  data() {
+    return {
+      addingDealership: false,
+      selectedDealership: null,
+      toast: { contant: "static", title: "Toast Title" },
+      tableItems: [],
+      tableFields: [
+        {
+          key: "name",
+          label: "Name",
+        },
+        {
+          key: "description",
+          label: "Description",
+        },
+        {
+          key: "admin",
+          label: "Owner",
+        },
+        {
+          key: "created_at_formatted",
+          label: "Date Created",
+        },
+      ],
+    };
+  },
+  mounted() {
+    this.fetchDealerships();
+  },
+  methods: {
+    clickRow(dealership) {
+      //this.selectedDealership = dealership
+      //console.log(dealership)
+      const dealershipId = dealership._id;
+      this.$router.push(`/dealerships/${dealershipId}`);
+    },
+    resetSelectedDealership() {
+      this.selectedDealership = null;
+      console.log("Selected Dealership has been reset");
+    },
+    fetchDealerships() {
+      axios({
+        method: "GET",
+        url: `${this.$store.state.api}/dealerships/?admin=${this.$store.state.auth.userId}&populate=admin`,
+        headers: {
+          Authorization: "Bearer " + this.$store.state.auth.token,
+        },
+      })
+        .then((response) => {
+          this.tableItems = response.data.payload;
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    },
+  },
+};
+</script>
+
+<style>
+</style>
```

Comment:
Empty tags

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/37/commits/8c83bea65e9c8a6186a069f6415340392a55a7b4)